nmma/em/analysis.py

```diff
@@ -744,6 +744,8 @@ def analysis(args):
 
     if args.bestfit:
         bestfit_to_write = bestfit_params.copy()
+        bestfit_to_write["ln_bayes_factor"] = round(result.log_bayes_factor, 3)
```

Comment:
might be better to maintain the 'log' convention even if we mean natural log, as the keys for the likelihood are 'log_likelihood' (unless I'm forgetting and the likelihood is log base 10 and bayes factor is natual log)